### ISSUE TITLE
Model training — baseline and tree-based models

### ISSUE THREAD

**Issue by FrcJules** [FrcJules/ca683-traffic-congestion#6]
## Objectif
Entraîner plusieurs modèles ML avec stratégie de split temporel.

## Modèles
- [x] Linear Regression (baseline)
- [x] Ridge Regression
- [x] Random Forest (n=100, max_depth=10)
- [x] Gradient Boosting (n=100, lr=0.1)
- [x] XGBoost (optionnel)
- [x] LightGBM (optionnel)

## Stratégie
Train: Jan–Mai 2023 | Test: Juin 2023 (split temporel)

Closes with commit cf657c6
